test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -207,25 +316,39 @@ def func(x):
 
         _run_test_with_fx_to_onnx_exporter_and_onnx_runtime(self, func, (tensor_x,))
 
+    @pytorch_test_common.skip_fx_exporters(
+        {
+            DynamoExporter: (
+                "AssertionError: Dynamo input/output is not consistent with traced input/output. "
+                "Ref: https://github.com/pytorch/pytorch/issues/96379"
```

Comment:
I think the status of `test_arange` and `test_scalar_tensor` should be updated with this PR.

Reply:
Updated.